### ISSUE TITLE
writeRecord and variable type

### ISSUE THREAD

**Issue by DierteK** [luads/php-xbase#87]
Hello,

I want to edit a DBF file and have created the following test script for this purpose:

```php
[...]
$table = new \Xbase\WritableTable($filename, null, 'CP1252');
$table->openWrite();
[...]

$record->foobar = 111.11;
$table->writeRecord();

```
The problem now is that a `string` was written in the field "foobar". But I would like to write something there as `float`. 

Thanks

**Comment by gam6itko** [comment]
@DierteK give me the file or `$table->version` value to reproduce this bug

**Comment by gam6itko** [comment]
I guess it fixed in #92
